set.mm

```diff
@@ -101409,8 +101409,26 @@ a Cantor normal form (and injectivity, together with coherence
   ${
     $d R w z $.  $d A w z $.
     $( Lemma for general well-founded recursion.  Establish a subset
-       relationship.  (Contributed by Scott Fenton, 11-Sep-2023.) $)
+       relationship.  (Contributed by Scott Fenton, 11-Sep-2023.)  (Revised by
```

Comment:
Note the reason for this revision (Transitivity revisited, for example)

Reply:
Done